tools/ci-build/build-image.dockerfile

```diff
@@ -0,0 +1,124 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+ARG base_image=public.ecr.aws/amazonlinux/amazonlinux:2
+FROM ${base_image} AS bare_base_image
+
+#
+# Node Installation Stage
+#
+FROM bare_base_image AS install_node
+ARG node_version=v16.14.0
+ARG node_bundle_sha256=0570b9354959f651b814e56a4ce98d4a067bf2385b9a0e6be075739bc65b0fae
```

Comment:
This is the SHA of the `node.tar.xz` for the version on the line above right?

Reply:
Yeah